### PR TITLE
Update (NS14.2.6) + Fix(import_config_url) + New(MONGODB_URI config option)

### PR DESCRIPTION
**Pull request by loopy321**
I have only built the amd64 image as I don't have a great setup for aarch64.
Testing and working on my system using either the local mongodb or via the new URI config option.
Thanks!

### CODE REVIEW

File: nightscout/Dockerfile [hunk 1]
```diff
@@ -67,8 +69,8 @@ LABEL \
     org.opencontainers.image.vendor="Marcio Granzotto" \
     org.opencontainers.image.authors="Marcio Granzotto <oscensores@gmail.com>" \
     org.opencontainers.image.licenses="GNUv3" \
-    org.opencontainers.image.source="https://github.com/marciogranzotto/addon-nightscout" \
-    org.opencontainers.image.documentation="https://github.com/marciogranzotto/addon-nightscout/blob/master/README.md" \
+    org.opencontainers.image.source="https://github.com/loopy321/addon-nightscout" \
+    org.opencontainers.image.documentation="https://github.com/loopy321/addon-nightscout/blob/master/README.md" \
```
Comment (marciogranzotto) [inline nightscout/Dockerfile:73]: please roll this back so I can merge it

File: nightscout/config.json [hunk 2]
```diff
@@ -1,14 +1,15 @@
 {
   "name": "Nightscout",
-  "version": "dev",
+  "version": "1.3.6",
   "slug": "nightscout",
   "description": "Nightscout acts as a web-based CGM (Continuous Glucose Montinor) to allow multiple caregivers to remotely view a patients glucose data in realtime.",
-  "url": "https://github.com/marciogranzotto/addon-nightscout",
+  "url": "https://github.com/loopy321/addon-nightscout",
```
Comment (marciogranzotto) [inline nightscout/config.json:6]: and this

File: nightscout/config.json [hunk 3]
```diff
@@ -1,14 +1,15 @@
...
   "arch": ["aarch64", "amd64"],
   "webui": "[PROTO:ssl]://[HOST]:[PORT:5423]",
   "startup": "services",
+  "init": "false",
   "boot": "auto",
-  "image": "marciogranzotto/image-{arch}-addon-nightscout",
+  "image": "loopy321/image-{arch}-addon-nightscout",
```
Comment (marciogranzotto) [inline nightscout/config.json:12]: and this too

**Comment by loopy321** [comment]
See new pull https://github.com/marciogranzotto/addon-nightscout/pull/16
with the addressed reversions.

**Pull request closed without merging**